router/core/router.go

```diff
@@ -1097,9 +1100,34 @@ func (r *Router) newServer(ctx context.Context, routerConfig *nodev1.RouterConfi
 			h.ServeHTTP(w, r)
 		})
 	})
+	var customCompressibleContentTypes = []string{
```

Comment:
This should be defined in the corresponding package.

Reply:
Package defaults does not contain graphql types and if we pass contain types then package will only use that types instead of default. In our case we need both so we have pass all types including graphql types.

Reply:
My point is that the router module should not host those variables. It would be good, if you put those variables into a dedicated package.

Reply:
Sure no problem